testnet/etcd/register_etcd.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
```

Comment:
Use `env` for shebangs.
```suggestion
#!/usr/bin/env bash
```
Also, why are we using `sh` and not `bash? This applies to all scripts.

Reply:
Also, if you're using bash use double square brackets in conditionals.

Reply:
The node uses `mindeb` image as base and that only contains `sh`. We couldn't use alpine because of musl and waku-rust-bindings + go-waku incompatibility (https://github.com/logos-co/nomos-node/pull/60#issue-1565003676), but I'll revisit this option with `libp2p` and `mixnet` images.

Reply:
I see, I thought we were using that `rust:1.70.0-slim-bullseye` image, but I see now there's a stage using `bitnami/minideb:latest`. Very well then.